packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/helpers/DevLauncherURLHelper.kt

```diff
@@ -0,0 +1,15 @@
+package expo.modules.devlauncher.helpers
+
+import android.net.Uri
+
+fun changeUrlSchema(uri: Uri, scheme: String): Uri = uri.buildUpon().scheme(scheme).build()
```

Comment:
I think `scheme` is more appropriate than `schema` and you used `scheme` everywhere else 😉 
```suggestion
fun changeUrlScheme(uri: Uri, scheme: String): Uri = uri.buildUpon().scheme(scheme).build()
```